src/imginputfileconn.h

```diff
@@ -594,6 +612,7 @@ namespace dd
     int _scale_min = 600;
     int _scale_max = 1000;
     bool _keep_orig = false;
+    std::string _interp = "cubic";
```

Comment:
same as cuda_resize : i think this _interp should be copyed at copy_contruction time